examples/chat_types_filter.py

```diff
@@ -0,0 +1,30 @@
+"""
+This is an example with usage of ChatTypesFilter
+It filters incoming object based on type of its chat type
+"""
+
+import logging
+
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.types import ChatType
+
+API_TOKEN = 'BOT TOKEN HERE'
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+
+# Initialize bot and dispatcher
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot)
+
+
+@dp.message_handler(chat_types=[ChatType.PRIVATE, ChatType.CHANNEL])
```

Comment:
Typo here, correct to use is chat_type= without 's'

Reply:
You’ve viewed the outdated version. Yes, the correct one is `chat_type`

Reply:
I see. Btw in other part of code too not work. I'll find and mark you there or open issue.